src/systems/selection/select_multiple_sprites.rs

```diff
@@ -1 +1,15 @@
-// TODO add system
+use bevy::prelude::EventReader;
+
+use crate::events::selection_events::SelectionAreaEvent;
+
+// TODO finish this off
+pub fn select_multiple_sprites(mut selection_area_reader: EventReader<SelectionAreaEvent>) {
+    let Some(_selection_area) = selection_area_reader.read().last() else {
+        return;
+    };
+
+    // TODO at this point the selection box has despawned (approximately) with the area the box sent in this event
```

Comment:
@MatthewGriffin should just be able to use the area in the "SelectionAreaEvent" along with the Starship's transforms to determine what is selected.

Then assign the SpawnSelection resource to MultipleSelections.